docs/src/main/asciidoc/_admin-guide/management-tasks/Command_line_parameters.adoc

```diff
@@ -196,6 +196,13 @@ be relative to jboss.server.config.dir. This is similar to
 --server-config but if this alternative is specified the server willnot
 overwrite the file when the management model is changed. However a full
 versioned history is maintained of the file.
+
+|--git-repo |- |A remote Git repository URL (e.g. https://github.com/wildfly/wildfly-config.git)
```

Comment:
A remote Git repository URL to use for configuration directory and content repository content or `local` if only a local repository is to be used.

(I added a phrase to provide context and you don't need an example git url. If they don't know what a git url is they're not a likely user.)